backend/app/tests/conftest.py

```
@@ -79,7 +79,7 @@ def client() -> Generator[TestClient, None, None]:
 
 
 @pytest.fixture(scope="module")
-def superuser_token_headers(client: TestClient) -> dict[str, str]:
+def superuser_token_headers(client: TestClient, db: Session) -> dict[str, str]:
```

Ruff (0.11.9)

Unused function argument: `db`

(ARG001)